instat/sdgPlots.vb

```diff
@@ -24,7 +24,7 @@ Public Class sdgPlots
     Public clsRsyntax As New RSyntax
     'This clsRSyntax is linked with the ucrBase.clsRSyntax from the dlg calling sdgPLotOptions...
     Public clsRggplotFunction As New RFunction
-    Public clsAesFunction As New RFunction    'Warning: I m not sure this field is useful... Will all be revised when changing links though...
+    Public clsGlobalAesFunction As New RFunction

```

Comment:
Added the global aes so that the sub dialog can pass through the column type for x and y to the axes controls